deno.jsonc

```diff
@@ -2,7 +2,7 @@
     "lock": false,
     "tasks": {
         "check": "deno cache --check=all src/mod.ts",
-        "backport": "deno run --no-prompt --allow-read=. --allow-write=. https://deno.land/x/deno2node@v1.7.1/src/cli.ts tsconfig.json",
+        "backport": "deno run --no-prompt --allow-read=. --allow-write=. https://lib.deno.dev/x/deno2node@v1/src/cli.ts tsconfig.json",
```

Comment:
We should pin the version of our build tooling. Otherwise, it may randomly break our setup.